dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/source/p/pandas/pandas-$portVersion.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
```

Comment:
@bach5000: For some reason `$python setup.py build` is crashing on my system. So I commented out the block which adds `python2.7` to see if `python3.6` would work better. I did not. `setup.py` also makes `python3.6` crash.

I noticed that `pandas-0.23.4` was out, so gave it a try. Same results, for both versions of Python.
Maybe my Haiku/x86_64 system is a bit broken. I did not update it since more than one month...

Is pandas building fine for your?
Anyway, in case you would like to switch from `0.23.3` to `0.23.4` here's the new checksum, for your convenience:
```diff
-CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+CHECKSUM_SHA256="5b24ca47acf69222e82530e89111dd9d14f9b970ab2cd3a1c2c78f0c4fbba4f4"
```

The recipe looks OK but I can't merge it because I would need to investigate why it is not building on my system.
Maybe another mentor can build the recipe and merge it.

CC @Begasus, @extrowerk, @alaviss

Reply:
@fbrosson It is building fine on my system (Haiku beta on x86_64). it just takes a really, really long time (and there are occasional errors and warning messages along the way, and the process seems to be stuck at some points, but it actually still runs if you wait long enough)...

But yes, I can update the version from `0.23.3` to `0.23.4`. Let's just hope no new bugs crawl out of the software... :smile: